openjdk-11.yaml

```diff
@@ -112,6 +112,43 @@ subpackages:
       runtime:
         - freetype
         - fontconfig
+  - name: openjdk-11-jre-headless
+    description: "Oracle OpenJDK 17 (JRE headless)"
+    pipeline:
+      - runs: |
+          export _FROM_ROOT=${{targets.destdir}}/usr/lib/jvm/openjdk
+          export _TO_ROOT=${{targets.subpkgdir}}/usr/lib/jvm/openjdk
```

Comment:
the JRE subpackage above is using jre in the folder path `/usr/lib/jvm/openjdk-jre` we should probably do the same here